src/librustdoc/html/render/print_item.rs

```diff
@@ -323,10 +323,10 @@ fn item_module(w: &mut Buffer, cx: &Context<'_>, item: &clean::Item, items: &[cl
 
                 write!(
                     w,
-                    "<tr class=\"{stab}{add}import-item\">\
-                         <td><code>{vis}{imp}</code></td>\
-                         <td class=\"docblock-short\">{stab_tags}</td>\
-                     </tr>",
+                    "<item-left class=\"{stab}{add}import-item\">\
+                         <code>{vis}{imp}</code>\
+                     </item-left>\
+                     <item-right class=\"docblock-short\">{stab_tags}</item-right>\n",
```

Comment:
Why the `\n`?

Reply:
Primarily, to make the output more readable and grepable. But I am open to removing it

Reply:
Please remove it.

Reply:
Done